data/SwapnilPatle01.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Swapnil Patle",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Front End Developer.Learner,Youtuber",
+    "avatar": "https://github.com/SwapnilPatle01.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SwapnilPatle01",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on YouTube",
+      "url": "https://www.youtube.com/channel/UCn0F3wdlhYyS3G5mO8li8Jg",
+      "icon": "FaYouTube"
+    },
+     {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/swappy___01/",
+      "icon": "FaInstagram"
+    } ,
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/swapnil-patle-821534223/",
+      "icon": "FaLinkedin"
+    },
+     {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/mdswapnil.patle",
+      "icon": "FaInstagram"
+    },
+     {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/SwapnilPatle01",
+      "icon": "FaGithub"
+    }
```

Comment:
This is an Indentation error, you just need to fix the spacings here
remove the empty line and fix the spacings

Reply:
I'd suggest you use vs-code or a [json validator](https://jsonformatter.curiousconcept.com/)